file-formats/uipg/type/zif_aff_uipg_v1.intf.abap

```diff
@@ -25,8 +25,8 @@ INTERFACE zif_aff_uipg_v1
 
   "! <p class="shorttext">Tile Display Format</p>
   "! Tile display format
-  "! $values {@link zif_aff_uipg_v1.data:co_tile_display_format}
-  "! $default {@link zif_aff_uipg_v1.data:co_tile_display_format.standard}
+  "! $values {@link /ui2/if_aff_uipg_v2.data:co_tile_display_format}
```

Comment:
Here you are referencing to data/types that are not in the repository, right? This most probably confused the test.

Reply:
Which ist ok, then the file format shall be self contained.

Reply:
that's what you get from not running through the entire generation process again. I forgot to adjust the the interface... unfortunately, the check is still red.

The Error Description is:
Error: UIPG: Provided and generated JSON Schema differ
Error:                   "maxLength": 2
Error:                   "maxLength": 2
What could this mean?

Reply:
The generation process knows only things that are in the repository.
`zif_aff_uipq_v1.data:co_tile_display_format` is inside the repository, `/ui2/if_aff_uipg_v2.data:co_tile_display_format` is not. Maybe stay with `zif_aff_uipq_v1.data:co_tile_display_format`? The generation process fails and yeah, the error message/description itself is not that helpfull.